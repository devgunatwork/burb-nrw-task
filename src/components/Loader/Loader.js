import React from 'react';
import LinearProgress from '@material-ui/core/LinearProgress';

const Loader = () => <LinearProgress color="secondary" />;

export default Loader;
