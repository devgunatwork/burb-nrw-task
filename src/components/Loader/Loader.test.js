import React from 'react';
import Loader from './Loader';

describe('components', () => {
  describe('Loader', () => {
    describe('UI', () => {
      it('should match the snapshot', () => {
        const wrapper = shallow(<Loader />);

        expect(wrapper).toMatchSnapshot();
      });
    });
  });
});
