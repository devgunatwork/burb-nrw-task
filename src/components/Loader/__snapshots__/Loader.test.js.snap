// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Loader UI should match the snapshot 1`] = `
<WithStyles(ForwardRef(LinearProgress))
  color="secondary"
/>
`;
